lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -44,7 +44,11 @@ def routable_request?(env)
     end
 
     def route(env)
-      Application.get_route(env[PATH_INFO], env[REQUEST_METHOD])
+      Application.get_route(env)
+    end
+
+    def span_name(env)
+      "#{env[REQUEST_METHOD].to_s.downcase} #{route(env)}".strip
```

Comment:
Since this `span_name` method will be called before making requests,
considerable refactoring seems to be needed to have "not_found" or "redirected" in span name.
I think it's better not to include the changes in this PR.